src/dashboard/install/archivematica-dashboard.service

```diff
@@ -1,4 +1,4 @@
-description "Archivematica Storage Service"
+description "Archivematica Dashboard"
```

Comment:
Good catch. But I don't understand why you've renamed this file, it's an upstart config file and their standard extension is `.conf`.

Reply:
It's a systemd service file, default extension is .service.   It is described in the readme.md as a systemd config file, not upstart, and it is being used as such by am-packbuild and the archivematica ansible role, isn't it?

Reply:
systemd units have a different format AFAIK. It's possible that am-packbuild provides its own systemd files. @scollazo 🏓

Reply:
ok, I see the format of this file is not systemd.  However the readme refers to it as a systemd example - either the syntax of this file should be updated to provide valid systemd synax, or the name of the file should be reverted to .conf and the readme updated to refer to it as an upstart example.

I think it should be systemd , so I will provide a new commit that updates the contents of the file.